### PR TITLE
[IMPROVEMENT] Refactored main() function.

### PR DESCRIPTION
**Pull request by alwayslivid**
**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/SwagLyrics/SwagLyrics-For-Spotify/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used the project.
- [x] I have used the project briefly.
- [ ] I have used the project extensively, but have not contributed previously.
- [ ] I am an active contributor to the project.

---

Performing checks in situations where the arguments won't end up going through degrade the overall user experience, and especially in situations where the user just wants to check out the commands or is experimenting with it.

My small, proposed change is meant to get rid of this unnecessary delay.

### CODE REVIEW

**Comment by aadibajpai** [comment]
Seems fair, the only reason I hadn't done this till now was since `swaglyrics -h` evades the precheck. Can you take a look over what pep8 is saying? I don't see anything wrong with it though.

**Comment by alwayslivid** [comment]
Me neither, and to be honest, that seems very confusing to me, provided that without the indentation that pep8 is complaining about, this change wouldn't work. @aadibajpai 

TL;DR It's equally confusing to me.
